scripts/run.py

```diff
@@ -0,0 +1,487 @@
+#!/usr/bin/python3
+
+import argparse
+import json
+import os
+import shutil
+import sys
+from fabric.api import task, local, abort, lcd, get, settings
+from fabric.contrib.console import confirm
+from fabric.contrib import files
+import csv
+import subprocess
+from termcolor import colored
+import datetime
+
+class Logger:
+    def log(self, msg, color='magenta'):
+        timestamp = colored("{:%H:%M:%S.%f}".format(datetime.datetime.now()), 'yellow')
+        label = colored("Runner", color)
+        print("%s[%s]%s" % (timestamp, label, msg))
+
+    def error(self, msg):
+        self.log(msg, color='red')
+
+    def raw(self, stdout, stderr):
+        sys.stdout.write(stdout + '\n')
+        sys.stderr.write(stderr + '\n')
+
+class Config:
+    def __init__(self, config_file, logger):
+        self.config = self.baseconfig(logger) if not config_file or config_file == "None" else self.load_config(config_file, logger)
+        self.logger = logger
+
+    def get(self, key):
+        if key not in self.config:
+            self.logger.error("[Config] Key %s not found" % key)
+            return None
+        else:
+            return self.config[key]
+
+    def baseconfig(self, logger):
```

Comment:
If I could choose I would get rid of the config - unless I miss something where it is really being used.
The things that should be configurable can be done using cli args or shell env.